lichess-bot.py

```diff
@@ -393,7 +393,7 @@ def setup_board(game):
         try:
             board.push_uci(move)
         except ValueError as e:
-            logger.debug('Ignoring illegal move {} on board {} ({})'.format(move, board.fen(), e))
+            logger.debug("    Ignoring illegal move {} on board {} ({})".format(move, board.fen(), e))
```

Comment:
That’s a surprising error! Might have something to do with this. I’ll revert it and check whether it still fails?